src/client/components/ChangesetDelta/ChangesetDelta.js

```diff
@@ -1,22 +1,42 @@
 /* @flow */
 import React from 'react'
+import PieChart from 'react-simple-pie-chart'
```

Comment:
Bar chart takes lots of space for the list of files that showed the left side of the diff component, but let's could play with different options, up to just removing this at all ...